ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/AuthEndpoint.java

```diff
@@ -230,6 +239,7 @@ public Response callback(
             .queryParam("state", session.state())
             .build();
 
+    authSuccesses.increment();
```

Comment:
A better term here might be completed. I have not read very deeply into the code, so I might have missed a path on which we might arrive here without the user actually having a from their point of view, successful result.

Just let me know if we should change the name here.

Reply:
This is the very start of the authentication flow here. Fine for now.